content/courses/intro-to-spark/chapters/05_Proof_Of_Functional_Correctness.rst

```diff
@@ -644,10 +649,19 @@ property.
 Ghost Procedures
 ~~~~~~~~~~~~~~~~
 
+.. todo::
+    
+    This section still seems very abstract. Can a more concrete example be developed?
```

Comment:
There is an example below of ghost procedures, can you explain why it's not sufficient?

Reply:
The thing that is missing here for me, is the connection between the Ghost procedure and the production code.  In the example presented I don't understand when I would want to use the Ghost variable ```A``` and the Ghost procedure ```Increase_A```. 

I think some of the later examples (like example 3) make this link. However, as a first time reader, I found myself re-reading this particular section several times.

Reply:
indeed, this section only presents the technique of ghost procedures, not their usage. Feel free to suggest reorganisation, although we want to keep exercises at the end of each section.